NAMING_CONVENTIONS.md

```diff
@@ -0,0 +1,37 @@
+Arnold USD naming conventions
+=======================
+
+The Arnold USD implementation supports arnold-specific attributes through custom schemas.
+
+## Arnold attributes on USD nodes
+
+Additional arnold attributes on USD-native nodes (geometries, lights, etc...) need to be in the "arnold" namespace
+
+Example:
+def DomeLight "my_light"
+{
+ float intensity=2
+ int arnold:max_bounces=99
+}
+
```

Comment:
We should document that primvars named `primvars:arnold:xxx` (ie `primvars:arnold:max_bounces`) are going to be translated to built-in parameters, rather than as user attributes.

Reply:
ok, I need to add support for this in the usd procedural then

Reply:
Let's make a separate ticket for that.

Reply:
created #43